docs/build/guides/building-a-voting-dapp.md

```diff
@@ -1036,7 +1047,7 @@ Lastly, to get the result of a transaction we are using the `pollTransactions` h
 To run the frontend dApp, go to the frontend folder and run:
 
 ```shell
-npm run start
+pnpm run dev
```

Comment:
People should be free to use any package manager of choice, so I guess it's best to use what everyone already knows in generic community documentation. I.e. users of Yarn, pnpm, bun etc. etc. know how to translate npm commands.

Reply:
Of course, but in this case we provide an example repository with code and have to make a choice there. This documentation just reflects that. It's not meant to tell people what to use themselves.

`npm run dev` would also work in a pnpm workspace though, so this could be changed.